### PR TITLE
Generalizing Django-Todo using settings.AUTH_USER_MODEL + small bug fix

### PR DESCRIPTION
**Pull request by shantisuresh01**
Hi Scott,

It is fun using the Django-Todo program!  Please review my suggested changes.

Thanks,

                  -Shanti

### CODE REVIEW

File: todo/forms.py [hunk 1]
```diff
@@ -24,7 +26,7 @@ def __init__(self, task_list, *args, **kwargs):
         super(AddItemForm, self).__init__(*args, **kwargs)
         # print dir(self.fields['list'])
         # print self.fields['list'].initial
-        self.fields['assigned_to'].queryset = User.objects.filter(groups__in=[task_list.group])
+        self.fields['assigned_to'].queryset = get_user_model().objects.filter(groups__in=[task_list.group])
```
Comment (shacker) [inline todo/forms.py]: This is a nice improvement, thanks.

File: todo/__init__.py [hunk 2]
```diff
@@ -1,10 +0,0 @@
-"""
```
Comment (shacker) [inline todo/__init__.py]: Oops - you've accidentally wiped out the __init__.py - I can no longer install the app. Please restore this file.

**Comment by shantisuresh01** [comment]
Ouch!

Will do!   Sorry about that.  I was messing with it as part of finding out
some issues.   Please stay tuned....

Thanks,


Best Regards,

                        -Shanti

**Comment by shantisuresh01** [comment]
Hi Scot,

I have restored __init__.py.  Please check.

Thanks,

                  -Shanti

**Comment by shacker** [comment]
Thanks for restoring the __init__ . All looks good. Thanks for your contribution!

**Pull request merged**